lib/rucio/client/commands/bin_legacy/rucio.py

```diff
@@ -1993,15 +1993,18 @@ def add_lifetime_exception(args, client, logger, console, spinner):
     dids_list = []
     containers = []
     datasets = []
-    error_types = ['Total DIDs',
-                   'DID not submitted because it is a file',
-                   'DID that are containers and were resolved',
-                   'DID not submitted because it is not part of the lifetime campaign',
-                   'DID successfully submitted including the one from containers resolved']
+    error_type_mapping = {
+        "total_dids": "Total DIDs",
+        "files_ignored": "DID not submitted because it is a file",
+        "containers_resolved": "DID that are containers and were resolved",
+        "not_in_lifetime_model": "DID not submitted because it is not part of the lifetime campaign",
+        "successfully_submitted": "DID successfully submitted including the one from containers resolved"
+    }
     for did in dids:
         scope, name = get_scope(did, client)
         dids_list.append({'scope': scope, 'name': name})
-    summary = {0: len(dids_list), 1: 0, 2: 0, 3: 0, 4: 0}
+    summary = {key: 0 for key in error_type_mapping}  # Initialize all summary counts
+    summary["total_dids"] = len(dids_list)
```

Comment:
instead of initializing `total_dids` to 0 first and then setting it to `len(dids_list)`, it can be initialized directly as `len(dids_list)`